src/SFA.DAS.Admin.Aan.Web/Extensions/DateTimeExtensions.cs

```diff
@@ -8,4 +8,12 @@ public static class DateTimeExtensions
 
     private readonly static TimeZoneInfo LocalTimeZone = TimeZoneInfo.FindSystemTimeZoneById("GMT Standard Time");
     public static DateTime UtcToLocalTime(this DateTime date) => TimeZoneInfo.ConvertTimeFromUtc(date, LocalTimeZone);
+
+    public static DateTime LocalToUtcTime(int year, int month, int day, int hour, int minutes)
```

Comment:
This is not an extension method, but I cannot think of a proper place to put this in, so I guess leave it.